.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+source = pybamm
+concurrency = multiprocessing
```

Comment:
We can use the `--concurrency` flag while executing coverage. Something like `--concurrency=multiprocessing`. We can then get rid of this file.

Reply:
Actually, no we can't—we can specify `--concurrency=multiprocessing` as a command-line argument but not `--source = pybamm`. The `coverage` docs say it needs this in a config file which can be either `tox.ini` or `setup.cfg` or `.coveragerc` (as discussed above). So it would be better to have both of these options in one place/file, in that sense

Reply:
Ah, we can do `--source` but we can't do `--source` and `--concurrency` together.

Reply:
Yes. I suppose an extra file should not be a problem until `coverage` adds support for `noxfile.py`